test/math.test.ts

```diff
@@ -104,48 +102,47 @@ test('rounding a date to the closest minute', () => {
 });
 
 describe('test adding days to date on different time-zones', () => {
-  const testTimezone = (tz: any) =>
-    describe('timezone: ' + tz, () => {
-      beforeAll(() => {
-        timezoneMock.register(tz);
-      });
-      afterAll(() => {
-        timezoneMock.unregister();
-      });
-
-      test('normal date', () => {
-        const date1 = new Date(2015, 10, 1, 0, 0, 0);
-        expect(date1.getDate()).toBe(1);
-
-        const date2 = dateByAddingDays(date1, 1);
-        expect(date2.getDate()).toBe(2);
-
-        const date3 = dateByAddingDays(date1, -1);
-        expect(date3.getDate()).toBe(31);
-      });
-
-      test('special date 1', () => {
-        const date1 = new Date(1667617200000);
-        expect(date1.getDate()).toBe(5);
-
-        const date2 = dateByAddingDays(date1, 1);
-        expect(date2.getDate()).toBe(6);
-
-        const date3 = dateByAddingDays(date1, -1);
-        expect(date3.getDate()).toBe(4);
-      });
-
-      test('special date 2', () => {
-        const date1 = new Date(1699066800000);
-        expect(date1.getDate()).toBe(4);
-
-        const date2 = dateByAddingDays(date1, 1);
-        expect(date2.getDate()).toBe(5);
-
-        const date3 = dateByAddingDays(date1, -1);
-        expect(date3.getDate()).toBe(3);
-      });
-    });
-
-  ['UTC', 'Australia/Adelaide', 'Brazil/East'].forEach(testTimezone);
+  test('normal date', () => {
+    const date1 = new Date(2015, 10, 1, 0, 0, 0);
+    expect(date1.getDate()).toBe(1);
+
+    const date2 = dateByAddingDays(date1, 1);
+    expect(date2.getDate()).toBe(2);
+
+    const date3 = dateByAddingDays(date1, -1);
+    expect(date3.getDate()).toBe(31);
+  });
+
+  test('special date 1', () => {
+    const date1 = new Date(1667617200000);
+    expect(date1.getDate()).toBe(5);
+
+    const date2 = dateByAddingDays(date1, 1);
+    expect(date2.getDate()).toBe(6);
+
+    const date3 = dateByAddingDays(date1, -1);
+    expect(date3.getDate()).toBe(4);
+  });
+
+  test('special date 2', () => {
+    const date1 = new Date(1699066800000);
+    expect(date1.getDate()).toBe(4);
+
+    const date2 = dateByAddingDays(date1, 1);
+    expect(date2.getDate()).toBe(5);
+
+    const date3 = dateByAddingDays(date1, -1);
+    expect(date3.getDate()).toBe(3);
+  });
+
+  test('special date 3', () => {
```

Comment:
please add comments to each of these special dates indicating which timezones they are testing

Reply:
added

Reply:
Please use IANA timezone names, but also Brazil no longer does DST as I mentioned previously so I'm not sure what we're testing in this scenario. I believe this test only failed when using the incorrect timezone-mock library.

Reply:
well I dont know what is IANA names, I used the "TZ database name" column of the wiki

Reply:
In the "TZ Database name" modern timezones are in the format "Continent/City". "Brazil/East" is considered deprecated and instead "America/Sao_Paulo" is used.

Reply:
these tests are not useful. `Date`'s constructor uses UTC timezone when you pass in a timestamp, which may be the inconsistency you are seeing.

Reply:
the problem is that it happens in a special environment, so probably you won't be able to test it in a normal environment

Reply:
at least with timezone-mock it was somewhat possible to test this